app/src/main/res/drawable-v24/ic_user_grey.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="512dp"
```

Comment:
please name this drawable more appropriately. We'll use syntax <relavantTag_color>